pkg/operator/ceph/object/config.go

```diff
@@ -156,6 +156,23 @@ func (c *clusterConfig) setFlagsMonConfigStore(rgwConfig *rgwConfig) error {
 
 	monStore := cephconfig.GetMonStore(c.context, c.clusterInfo)
 	who := generateCephXUser(rgwConfig.ResourceName)
+
+	configOptions, err := c.generateMonConfigOptions(rgwConfig)
+	if err != nil {
+		return err
+	}
+
+	for flag, val := range configOptions {
+		err := monStore.Set(who, flag, val)
```

Comment:
Can we use `monStore.SetAll()`? It would be nice to reduce the number of individual calls to set. This is needed in other places for rgw too, but for a separate PR.